src/main/java/core/basesyntax/model/machine/Car.java

```diff
@@ -1,9 +1,29 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Car extends Machine {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
This generationType will not work here - should be Generation.type.TABLE

Reply:
@fyarkiy Car and other child classes should not have id field at all
but if you meant the parent entity here - you are right